### PR TITLE
updated plugins.md docs

### PR DESCRIPTION
**Pull request by Plamen5kov**

### CODE REVIEW

**Comment by teobugslayer** [comment]
:-1:

File: PLUGINS.md [hunk 1]
```diff
@@ -33,11 +34,12 @@ The plugin must have the directory structure, described in the [Directory Struct
 If the NativeScript framework does not expose a native API that you need, you can develop a plugin which exposes the required functionality. When you develop a plugin, keep in mind the following requirements.
 
 * The plugin must be a valid npm package.
-* The plugin must expose a built-in native API or a native API available via custom native libraries.
+* The plugin must expose a built-in native API or a native API available via custom native libraries. (why "must")
```
Comment (teobugslayer) [inline PLUGINS.md]: The `(why "must")` text should not be part of the docs.

File: PLUGINS.md [hunk 2]
```diff
@@ -33,11 +34,12 @@ The plugin must have the directory structure, described in the [Directory Struct
...
 * The plugin must be written in JavaScript or TypeScript and must comply with the CommonJS specification. If written in TypeScript, make sure to include the compiled `JavaScript` file in your plugin.
 * The plugin directory structure must comply with the specification described below.
 * The plugin must contain a valid `package.json` which complies with the specification described below.
 * If the plugin requires any permissions, features or other configuration specifics, it must contain `AndroidManifest.xml` and `Info.plist` file which describe them.
+* (android speciffic) If the plugin has native dependencies, it must contain include.gralde file, which describe them. (suggestion)
```
Comment (teobugslayer) [inline PLUGINS.md]: Remove `(suggestion)`.

**Comment by ligaz** [comment]
Why :-1:? Can you give some info?

**Comment by teobugslayer** [comment]
Sorry, I forgot to remove my comment. It turned out that this is WIP and not ready for review yet.

**Comment by Plamen5kov** [comment]
i think it's ready for final review @ikoevska.

**Pull request merged**